.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/metal"]
```

Comment:
I don't think so, but I can imagine a time where we want to pin a given RAJA version on a specific metal version.  Specifically if there comes a time when we need to support a compiler they have stopped supporting.